pkg/scheduler/framework/session_plugins.go

```diff
@@ -443,7 +455,8 @@ func isEnabled(enabled *bool) bool {
 func (ssn *Session) NodeOrderMapFn(task *api.TaskInfo, node *api.NodeInfo) (map[string]float64, float64, error) {
 	nodeScoreMap := map[string]float64{}
 	var priorityScore float64
-	for _, tier := range ssn.Tiers {
+	Tiers := getTiersFromConfig(ssn.SchedStConf)
```

Comment:
lower case

Reply:
sorry , which one you want me to make lower case?

Reply:
`Tiers `